subsys/mgmt/Kconfig.mcumgr

```diff
@@ -22,11 +22,15 @@ config MGMT_CBORATTR_MAX_SIZE
 
 menu "Command Handlers"
 menuconfig MCUMGR_CMD_FS_MGMT
-	bool "Enable mcumgr handlers for file management"
+	bool "Enable mcumgr handlers for file management (insecure)"
 	depends on FILE_SYSTEM
 	help
 	  Enables mcumgr handlers for file management
 
+	  This option allows mucmgr access anything in file system
```

Comment:
This doesn't read well.  Something like this would be more clear.

This option allows mcumgr clients to access anything in the file system, including application-stored secrets like private keys.  Use of this feature in production is strongly discouraged.

Reply:
Indeed, thanks !